test/test-operation.js

```diff
@@ -32,260 +32,638 @@ var helpers = require('./helpers');
 var Sway = helpers.getSway();
 
 describe('Operation', function () {
-  var swaggerApi;
+  context('swaggerApiRelativeRefs', function () {
```

Comment:
That is VS Code with an Atom theme 🙂 
Sure, I can switch back to `describe` 👍 I just used it as I prefer `context` in my personal code because it reads a bit nicer.